lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Stemmer.java

```diff
@@ -138,6 +142,23 @@ WordCase caseOf(char[] word, int length) {
     return lowerBuffer;
   }
 
+  // Special prefix handling for Catalan, French, Italian:
```

Comment:
seems like this can be made static?

Reply:
I'll merge it in, you can piggyback static method on a different PR if you wish.